superset/models/helpers.py

```diff
@@ -171,7 +163,7 @@ def import_from_dict(
 
         # Remove fields that should not get imported
         for k in list(dict_rep):
-            if k not in export_fields:
+            if k not in export_fields and k not in parent_refs:
```

Comment:
This is necessary to preserve the parent ID in the config when importing, otherwise it gets deleted.